bin/.htmltest.yml

```
@@ -2,6 +2,7 @@ DirectoryPath: "public"
 EnforceHTTPS: false
 IgnoreURLs:
 - "/api/*"
+- "zero-trust/index.html"
```

```suggestion
- "/zero-trust/index.html"
```
Nit: Not sure if it should start with a slash, like `/api/*` does. I think we should at least be consistent (both `IgnoreURLs` and `IgnoreDirs` accept regexes).

htmltest also seems to support [tag-based ignores](https://github.com/wjdp/htmltest#see_no_evil-ignoring-content), if we wanted to ignore just a single URL. We'd have to use an HTML anchor element for this instead of Markdown, though.